### PR TITLE
Seperate requirements-dev.txt

### PR DESCRIPTION
**Pull request by gautamjajoo**
Removed dependencies needed to run only tasks from requirements.txt and moved to requirements-dev.txt.
@RishabhJain2018 @vkartik97

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+coverage==4.5.1
+coveralls==1.3.0
+flake8==3.0.4
+pytest==3.5.1
+pytest-cov==2.5.1
+requests-toolbelt==0.8.0
```
Comment (RishabhJain2018) [inline requirements-dev.txt]: @gautamjajoo I am not sure whether we need this package or not. Can you please install CLI without using this package & check if it works fine.

Comment (gautamjajoo) [reply requirements-dev.txt]: @RishabhJain2018 I tried doing that, but was not successful. I think so we need this package.

Comment (gautamjajoo) [reply requirements-dev.txt]: @RishabhJain2018 Yeah. It works fine without it.

**Comment by vkartik97** [comment]
LGTM :+1:
@RishabhJain2018  @deshraj  Please review

**Comment by RishabhJain2018** [comment]
Closing this since we don't need it.

**Pull request closed without merging**